hibernate-core/src/test/java/org/hibernate/orm/test/timezones/UTCNormalizedInstantTest.java

```diff
@@ -34,8 +37,8 @@ public class UTCNormalizedInstantTest {
 		});
 		scope.inSession( s-> {
 			Zoned z = s.find(Zoned.class, id);
-			assertEquals( instant, z.utcInstant );
-			assertEquals( instant, z.localInstant );
+			assertThat( z.utcInstant ).isCloseTo( instant, within( 1, ChronoUnit.MILLIS ) );
```

Comment:
Is this change for Sybase? If so, I think it would be nice to try using this instead `long millis = System.currentTimeMillis(); Instant instant = Instant.ofEpochMilli( millis - millis % 3 );` and add a comment that Sybase has a precision of 1/3 milliseconds.

Reply:
No this was failing on the pipeline with h2 on s390x, see [this example](https://ci.hibernate.org/job/hibernate-orm-pipeline/job/main/1740/testReport/junit/org.hibernate.orm.test.timezones/UTCNormalizedInstantTest/Build___s390x_h2___Test___test_SessionFactoryScope_/).

I noticed the extracted instant was always rounded to milliseconds, so that is what I tried to check. Don't know if there's a better way to handle this.

Reply:
Thanks for the pointer, I've updated the test with the rounding logic and removed the duplicated tests just for Sybase since they were no longer needed